docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3680,17 +3688,21 @@ does not need to keep track of group ids and member ids.
 
 One shots are programmed by choosing the `ActionProfileActionSet` message as the
 `TableAction`. The `ActionProfileActionSet` message consists of a set of
-`ActionProfileAction` messages, which in turn have the following fields:
+`ActionProfileAction` messages. This set should have cardinality no greater than
+`max_group_size` (if greater than 0) if `selector_size_semantics` is
```

Comment:
Can we replace "(if greater than 0)" with "(if max_group_size is nonzero)" to make the object explicit? 
I initial thought the implicit object was "cardinality".

Reply:
Done.